packages/block-editor/src/store/test/actions.js

```diff
@@ -809,23 +809,36 @@ describe( 'actions', () => {
 		} );
 
 		it( 'should only focus the blockA if the blockA has no merge function', () => {
```

Comment:
```suggestion
		it( 'should only focus the blockA if the blockA has no merge function and the content of blockB is modified', () => {
```

I think we should also change the title according to the updated test.